src/app/charts/line-chart/line-chart.component.ts

```diff
@@ -0,0 +1,49 @@
+import { Component, OnInit, ViewEncapsulation } from '@angular/core';
```

Comment:
is this code  generated? mb delete all tests and other redundant code to make project more clean and readable?